workers/unity/Packages/com.improbable.gdk.mobile/Editor/iOSLaunchUtils.cs

```diff
@@ -0,0 +1,292 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using System.Xml;
+using Improbable.Gdk.Tools;
+using UnityEditor;
+using UnityEngine;
+using Debug = UnityEngine.Debug;
+
+namespace Improbable.Gdk.Mobile
+{
+    public static class iOSLaunchUtils
+    {
+        private static readonly string XCodeProjectPath = Path.GetFullPath(Path.Combine(Application.dataPath, "..", "build", "worker", "MobileClient@iOS", "MobileClient@iOS"));
```

Comment:
I think we have a `UnityProjectRoot` path floating about somewhere 🤔

Reply:
yes, but only in the build system package. want me to move it into the tools package so all packages have access to it?

Reply:
That seems reasonable 😄